api/src/main/java/com/fit_goal/enums/Message.java

```diff
@@ -0,0 +1,20 @@
+package com.fit_goal.enums;
+
+public enum Message {
+
+    REGISTER_MESSAGE("Follow the link to confirm registration "),
```

Comment:
*_MESSAGE looks redundant, you have the enum Message, please keep the naming simple, but enough to describe what you wanted.

Reply:
refactored